common/workunit/workunit.cpp

```diff
@@ -1515,7 +1522,6 @@ class GraphScopeIterator : public CInterfaceOf<IConstWUScopeIterator>
             }
             if (whichProperties & PTstatistics)
             {
-                playAttribute(visitor, WaLabel);
```

Comment:
Cut and paste error.  Previously the label was returned twice by wudetails - it should not be included in the stats list.